less/mixins.less

```diff
@@ -1 +1,21 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+// .a-tag-pmixin(@textdecor,@tcolor, @fontfam){
+//   text-decoration: @textdecor;
+//   color: @tcolor;
+//   font-family: @fontfam
+// }
+// a{
+//   .a-tag-pmixin(none, black,'Roboto', sans-serif)
+// }
+.btn-pmixin (@bgColor: @button-bg, @height: 40px, @width: 80%, @fontAlign: center, @fSize: 1.5rem, @fontColor: @white, @borderR: 1rem){
+  background-color: @bgColor;
+  height: @height;
+  width: @width;
+  text-align: @fontAlign;
+  font-size: @fSize;
+  color: @fontColor;
+  border-radius: @borderR
+}
+button{
```

Comment:
right here what you're doing is assigning the mixin to all button elements.  Is that your intended effect?

Reply:
Even if it is, you should set this style inside the home-page.less file and keep this page solely for mixins.